nf_core/pipeline-template/main.nf

```diff
@@ -58,10 +57,10 @@ workflow {{ prefix_nodash|upper }}_{{ short_name|upper }} {
     {{ short_name|upper }} (
         samplesheet
     )
-
+{%- if multiqc %}{%- if modules %}
```

Comment:
doesn't `modules` include the setting `multiqc`, ie. if we disable modules, we disable multiqc, so we only need `multiqc` here?

Reply:
skipping modules deletes all multiqc files, but we manage it with the yaml file, not by setting multiqc to false

Reply:
fine to keep the logic simpler